client/transactions/declarations.d.ts

```diff
@@ -129,6 +129,12 @@ declare module '@woocommerce/navigation' {
 		type_is_not?: unknown;
 		source_device_is?: unknown;
 		source_device_is_not?: unknown;
+		channel_is?: unknown;
```

Comment:
Shouldn't the type for all the new fields be `string`? Perhaps our [TS guidelines](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/README.md) provide hints?

Reply:
Well spotted! I have updated the types.